backend/app/extraction/TranscriptionExtractor.scala

```diff
@@ -0,0 +1,78 @@
+package extraction
+
+import model.{English, Language, Languages}
+import model.manifest.Blob
+import org.apache.commons.io.FileUtils
+import services.{ScratchSpace, Tika}
+import services.index.Index
+import services.ingestion.IngestionServices
+import utils.{Logging, OcrStderrLogger, Whisper}
+import utils.attempt.AttemptAwait._
+import utils.attempt.{Failure, UnknownFailure, UnsupportedOperationFailure}
+import cats.syntax.either._
+
+import java.io.{File, InputStream}
+import scala.concurrent.ExecutionContext
+import scala.io.Source
+import scala.reflect.runtime.universe.Try
+import scala.util.control.NonFatal
+
+class TranscriptionExtractor(index: Index, scratchSpace: ScratchSpace, ingestionServices: IngestionServices) extends FileExtractor(scratchSpace) with Logging {
+  val mimeTypes: Set[String] = Set(
+    "audio/wav",
+    "audio/vnd.wave",
+  )
+
+  def canProcessMimeType: String => Boolean = mimeTypes.contains
+
+  override def indexing = true
+  override def priority = 5
```

Comment:
these (indexing/priority) are copy pasted from ocrmypdf and may need thinking about